.github/RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,78 @@
+<!--
+Please  use the issue title 'Release X.Y.Z preparation' and replace X, Y and Z
+with the major, minor and patch release version number.
+
+The X.Y.Z convention as placeholder for the actual version number is used
+thoughout this document.
+-->
+
+This issue is used to organize the version X.Y.Z release of FreeOrion.
+
+
+### Milestones
+
+Important milestones for the release.  Changes will be announced by the release
+manager.
+
+<!--
+Add all relevant milestones ordered by date or sequence.  Use the YYYY-mm-dd
+format for the date.  When a milestone entry is postponed copy the whole entry
+and add it below the original entry.  Use ~~ to strike out the date in the
+original entry and add a **POSTPONED** in front of the original milestone.
+
+The time references used are mere suggestions.  'SP' stands for starting point,
+the first milestone of the release.  '+1W', '+2W' should be interpreted as
+'one week after SP', 'two weeks after SP' and so on.
+-->
+
+| Date             | Milestone                         |
+|:---------------- | --------------------------------- |
+| <!-- SP  -->     | Create release branch from master |
+| <!-- +1W -->     | Release Candidate 1               |
+| ~~<!-- +2W -->~~ | **POSTPONED** Release Candidate 2 |
+| <!-- +3W -->     | Release Candidate 2               |
+| <!-- +3W -->     | Declaring Release                 |
+
+
+### Checklist
+
+This is a checklist for the release manager to ensure all steps are done before
+doing the actual release.
+
+* [ ] Create a new GitHub Issue from `.github/RELEASE_TEMPLATE.md` and follow
+      instructions. ;)
```

Comment:
Nice one. I'm seriously tempted to actually keep that, just for the smile it will give me every time... 😃